circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  ruby:
+    version: '2.3'
+
+dependencies:
+  cache_directories:
+    - 'vendor/bundle'
+  override:
+    - bundle -j8 --path=vendor/bundle
+
+database:
+  override:
+    - bundle exec rake db:create RAILS_ENV=test
+    - bundle exec rake db:migrate RAILS_ENV=test
```

Comment:
I think this line may not be necessary while this code automatically do the same thing on every run.

https://github.com/oishi-kenko/katgut/blob/master/spec/rails_helper.rb#L11